packages/components/src/VirtualizedList/RowCollapsiblePanel/RowCollapsiblePanel.component.js

```diff
@@ -62,7 +55,12 @@ class RowCollapsiblePanel extends React.Component {
 						aria-label={get(rowData, 'header[0].label')}
 						style={style}
 					>
-						<PureCollapsiblePanel {...{ rowData, measure, onToggle: this.onToggle }} />
+						<CollapsiblePanel
+							onEntered={measure}
```

Comment:
I rather name it onEnter

Reply:
It seems that `react-bootstrap` collapsible panel that our component use, has a prop `onEntered`. Let's not change the name to do a mapping in our component.